tests/struct.lisp

```diff
@@ -118,7 +118,7 @@
 
 (defctype s-s-ch (:struct s-s-ch))
 
-(defcvar "the_s_s_ch" s-s-ch)
+(defvar *the-s-s-ch* (foreign-symbol-pointer "the_s_s_ch"))
```

Comment:
Nice idea.

This has the drawback of not working with saved cores since it'll permanently store that pointer. Not a big deal for the test suite, but since this code might be used as an example, maybe change `defvar` do `define-symbol-macro`.

Alternatives:

1. Keep the `defcvar` and sprinkle `get-var-ptr` throughout.
2. Change the `defctype`s to something that doesn't translate structs, either with a custom type or by adding some option to the `:struct` type that skips translation. Dubious. Probably too much work.

I'm fine with your proposed solution (with `defvar` changed to `define-symbol-macro`).